src/NuGet.Clients/NuGet.VisualStudio/NuGet.VisualStudio.csproj

```diff
@@ -25,25 +35,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Update="install.ps1">
-      <CopyToOutputDirectory Condition=" '$(VisualStudioVersion)' == '15.0' ">PreserveNewest</CopyToOutputDirectory> <!-- TODO: Remove this when we stop building for VS14 -->
-    </None>
-  </ItemGroup>
-
-  <PropertyGroup>
-    <NuspecFile>NuGet.VisualStudio.nuspec</NuspecFile>
-    <NuspecProperties>version=$(Version);configuration=$(Configuration)</NuspecProperties>
-    <NuspecBasePath>$(OutputPath)</NuspecBasePath>
-  </PropertyGroup>
-
-  <!--
-  <ItemGroup>
-    <None Include="install.ps1">
-      <PackagePath>tools\</PackagePath>
+    <Content Include="NuGet.VisualStudio.targets">
+      <PackagePath>build</PackagePath>
```

Comment:
i am wondering if this needs to be build/TFM and buildCrossTargeting instead so it works for both inner and outer builds
@emgarten suggestions?

Reply:
Why would need it in the outer build?

Reply:
not needed, already discussed with justin